### PR TITLE
New Design System

### PR DESCRIPTION
**Pull request by Nyumat**
Finally had some time to flesh the rest of the UI revamp. It's looking good.

Summary of what's going on (yeah, there's a **lot** of changes)

**Server**
- Added a new script `dev` which allows hot reloading of the sever-side codebase.

**Client**
- Added a completely new design system, using `#0265AC` as the base 
- Incoproated the DRY (Don't Repeat Yourself) method for `App.js`
- Made new components in `src/components/landing` and `src/components/nav`
- Adjusted the styling globally to fit the design system

**TODO:**
- Apply the design system to the profile page
- Figure out if we **actually** need data fetching on the marketing page
- Clean up the code into custom hooks and actions
- Add a README

**Preview of UI/UX Changes**

## Hero
Hero

## Hero Mobile
Hero Mobile

## Offerings
Offerings

## Offerings Mobile
Offerings Mobile

## CTA
CTA

## CTA Mobile
CTA Mobile

### CODE REVIEW

File: client/.prettierrc [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+    "trailingComma": "es5",
+    "tabWidth": 4,
```
Comment (aidanouckama) [inline client/.prettierrc:3]: Unfortunately, the tab width set within the project was changed to 2 tabs, which is the cause of most of the conflicts in this merge 😭

File: client/src/App.js
Comment (aidanouckama) [inline client/src/App.js:1]: DRY 🙏

File: client/src/components/ConditionalLink.js
Comment (aidanouckama) [inline client/src/components/ConditionalLink.js:1]: Like this is an example of an unchanged file, just our Prettier settings didn't match 😔 Not sure if there's an easier way to fix than changing `client/.prettierrc` and manually saving each file... but you're the GOAT so maybe you know

**Review by aidanouckama — COMMENTED** [review]
I mean maybe we can just approve the merge and switch to these ESLint + Prettier settings (I like them more) but I'll leave it up to the team to decide today.

File: client/src/App.js [hunk 2]
```diff
@@ -1,64 +1,72 @@
...
-          <Route path="/user/:id" element={<Profile />} />
-
-          {/* Suggestions */}
-          <Route path="/Suggestions" element={<Suggestions />} />
-
-          {/* TESTING */}
-          {/* process.env.REACT_APP_NODE_ENV === "DEV" && <Route path="/test" element={<Test />} /> */}
+            {routes.map(({ path, element }, index) => (
+                <Route key={index} path={path} element={element} />
+            ))}
```
Comment (Lewin-B) [inline client/src/App.js:52]: Smart

**Review by aidanouckama — APPROVED** [review]
Let's see how this goes!

**Review by Lewin-B — APPROVED** [review]
Yeah most of these changes are formatting changes which are fine. And for some of the css i dont even know what I'm reading so i'm just gonna let you cook. Looks good to me

**Comment by aidanouckama** [comment]
Closing due to Netlify check bugs...

**Pull request closed without merging**